src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas1/Cas1SpaceBookingController.kt

```diff
@@ -161,12 +163,28 @@ class Cas1SpaceBookingController(
   ): ResponseEntity<Unit> {
     userAccessService.ensureCurrentUserHasPermission(UserPermission.CAS1_SPACE_BOOKING_RECORD_ARRIVAL)
 
+    val arrivalDateAndTime = if (cas1NewArrival.arrivalDateTime != null) {
+      Pair(
+        cas1NewArrival.arrivalDateTime!!.toLocalDate(),
+        cas1NewArrival.arrivalDateTime!!.toLocalDateTime().toLocalTime(),
```

Comment:
do we need the double bangs? Think the `if` above will negate the need

Reply:
So ... we shouldn't, but the command line compiler (i.e. not intellij) was convinced we do. This is only temporary code (cas1NewArrival.arrivalDateTime is deprecated) so I decided to just leave as-is instead of digging into why it was complaining. Good spot!